server/createServer.js

```diff
@@ -0,0 +1,20 @@
+const { GraphQLServer } = require('graphql-yoga');
+const db = require('./db');
+const Query = require('./src/resolvers/Query');
+const Mutation = require('./src/resolvers/Mutation');
```

Comment:
Can we use ES6 for importing modules? I think we need to integrate babel for that. Let's look it up